docs/concepts/manage-runs.md

```diff
@@ -1,4 +1,102 @@
 (manage-runs)=
 # {octicon}`terminal` Managing Runs
 
-Explain all the function of the run object (logs, status, etc.)
+After you have started a fine-tuning or a synthesis job by calling {meth}`~finetuner.fit` or {meth}`~finetuner.synthesize`,
+you get a {class}`~finetuner.run.Run` object in response.
+This run object serves as a handle for the cloud job that you triggered.
+If you lost the run object for the job, you can create a new `Run` object with the {meth}`finetuner.get_run` function:
+
+```python
+import finetuner
+
+finetuner.login()
+
+run = finetuner.get_run('your-run-name', 'your-experiment-name')
+```
+Since your runs are only accessible to you, you need to login first.
+If you did not specify an experiment name when calling the `fit` or `synthesize` function, 
+you can omit the experiment name in the `get_run` function as well.
+Further, you can use the {meth}`finetuner.list_runs` function to obtain a list of run objects for all jobs you submitted.
```

Comment:
```suggestion
Furthermore, you can use the {meth}`finetuner.list_runs` function to list the names and experiment names of all the jobs you have submitted. If you have lost all information about your run, you can look for it in that list.
```

Reply:
it does not list the names but it returns a list of run objects.